Sources/SwiftUICoordinator/Coordinator/Coordinator.swift

```diff
@@ -37,4 +40,8 @@ public extension Coordinator {
     func remove(coordinator: Coordinator) {
         childCoordinators.removeAll(where: { $0 === coordinator })
     }
+    
+    func handle(_ deepLink: DeepLink, with params: [String: String] = [:]) {
+        Logger.deepLink.warning("Deep link handler is not implemented.")
```

Comment:
maybe also have an assertion failure here, to crash with stack trace on debug builds. Might be more effective and won't get lost in apps with lots of logging

Reply:
Rather than stopping the program, I want to inform the user that the link will not be handled.
In my opinion, it's not so crucial to stop the app if the `handle` is called with the default implementation.

Reply:
After consideration, I agree that `assertionFailure` will help to catch errors early in the development process.